_posts/developers/pod-server/2019-01-02-00_apache-config.md

```diff
@@ -3,22 +3,26 @@ layout: for-developers
 title: "Using Apache as a reverse proxy"
 permalink: self-hosting/nss/apache
 tags: [pod-server]
-categories: [Running Node Solid Server]
+categories: [Running Community Solid Server]
 exclude: true
 redirect_from:
   - /for-developers/pod-server/apache
 ---
 
 # Using Apache as a Reverse Proxy
-
-* enable modules (ssl.conf, ssl.load, proxy.conf, proxy.load, proxy_html.conf, proxy_html.load, proxy_http.conf, proxy_http.load, rewrite.load, socache_shmcb.load)<br />
-* this is done by creating symlinks in `/etc/apache2/mods-enabled/` pointing to `/etc/apache2/mods-available/*`<br />
-* sample for one module other modules equivalent<br />
-`$ cd /etc/apache2/mods-enabled`<br />
-`$ ln -s ../mods-available/ssl.conf ssl.conf`<br />
-* edit 000-default.conf<br />
-`$ nano /etc/apache2/sites-available/000-default.conf`<br />
-add to config in the appropriate section as follows<br />
+A reverse proxy allows you to run a Solid server on a local port,
```

Comment:
I think the , is unnecessary(?) since its "A reverse proxy allows you to run .. and let..."?